GitHubRepositories.ps1

```diff
@@ -1241,6 +1236,18 @@ filter Set-GitHubRepository
     if ($PSBoundParameters.ContainsKey('IsTemplate')) { $hashBody['is_template'] = $IsTemplate.ToBool() }
     if ($PSBoundParameters.ContainsKey('Archived')) { $hashBody['archived'] = $Archived.ToBool() }
 
+    if ($Force -and (-not $Confirm))
+    {
+        $ConfirmPreference = 'None'
+    }
+
+    if (-not $PSCmdlet.ShouldProcess($RepositoryName, $shouldProcessMessage))
```

Comment:
We need to talk through this change a bit more ([previous conversation](https://github.com/microsoft/PowerShellForGitHub/pull/242#discussion_r441736038)).

`Rename-GitHubRepository` used to require confirmation.  In #242, this was refactored to just internally use `Set-GitHubRepository` which could already support that scenario.  Given that renaming a repository is one of those actions that is pretty destructive, it makes sense to me that it should require confirmation.  But we should only be requiring confirmation if the repo is being renamed.  So, I don't think that `ShouldProcess` should _always_ be happening...only if the repo is being renamed...but if it's being renamed, it should require confirmation.

Let's discuss further here.

Reply:
Scratch that.  I've just experimented with this change more, and now I understand it better.  It's working exactly the way I was desiring.

Brilliant.
Well done.